spec/lib/aliquot/payment_spec.rb

```diff
@@ -0,0 +1,3 @@
+describe Aliquot::Payment do
+  it 'works'
```

Comment:
I think the specs for the crypto stuff is 💯 important 🙂

Reply:
I agree on principle, however I was unsure about how to test, so i chose functional testing instead :)

I'll try and add some spec.s